From d9131efd623528b184045226c6ad8e949234dcd6 Mon Sep 17 00:00:00 2001
From: Christos Froussios <cfroussios@chromium.org>
Date: Fri, 15 Jul 2022 14:30:19 +0000
Subject: [PATCH 092/110] [OSCrypt] Fix branded GnomeKeyring tests

(cherry picked from commit 25dcc5ed790d7bbd56bcebe1a34f181c3716d7fb)

Bug: 1211311
Change-Id: I416600b84b8253f065ec4481b07552717cd7d8f5
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3738181
Commit-Queue: Lei Zhang <thestig@chromium.org>
Reviewed-by: Lei Zhang <thestig@chromium.org>
Auto-Submit: Christos Froussios <cfroussios@chromium.org>
Cr-Original-Commit-Position: refs/heads/main@{#1020094}
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3763527
Bot-Commit: Rubber Stamper <rubber-stamper@appspot.gserviceaccount.com>
Commit-Queue: Rubber Stamper <rubber-stamper@appspot.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/5005@{#1251}
Cr-Branched-From: 5b4d9450fee01f821b6400e947b3839727643a71-refs/heads/main@{#992738}
---
 components/os_crypt/key_storage_keyring_unittest.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/components/os_crypt/key_storage_keyring_unittest.cc b/components/os_crypt/key_storage_keyring_unittest.cc
index ae92edbb1f95..071a1bd5185c 100644
--- a/components/os_crypt/key_storage_keyring_unittest.cc
+++ b/components/os_crypt/key_storage_keyring_unittest.cc
@@ -130,7 +130,8 @@ class GnomeKeyringTest : public testing::Test {
 };
 
 GnomeKeyringTest::GnomeKeyringTest()
-    : task_runner_(new base::TestSimpleTaskRunner()), keyring_(task_runner_, "chromium") {
+    : task_runner_(new base::TestSimpleTaskRunner()),
+      keyring_(task_runner_, kApplicationName) {
   MockGnomeKeyringLoader::ResetForOSCrypt();
 }
 
-- 
2.38.1

